ros2-release/ros2-release.spec

```
@@ -0,0 +1,40 @@
+Name:           ros2-release
+Version:        1.0.0
+Summary:        Packages for ROS 2 main repository configuration
+Release: 1%{?dist}%{?release_suffix}
```

I think we should drop the `dist` suffix from the release here for a few reasons.
1. RPMFusion doesn't have it
2. The package is dirt simple and will work fine on any modern RHEL clone regardless of where it was built
3. It will make our installation instructions easier

```suggestion
Release: 1%{?release_suffix}
```